packages/acceptance-tests/pkg-tests-core/sources/utils/tests.ts

```diff
@@ -729,13 +729,15 @@ export const generatePkgDriver = ({
           // So add env variable TEST_IGNORE_TIMEOUT_FAILURES to turn on this suppression
           // TODO: investigate whether this is still needed.
           if (process.env.TEST_IGNORE_TIMEOUT_FAILURES) {
+            let timer;
             await Promise.race([
               new Promise(resolve => {
                 // Resolve 1s ahead of the jest timeout
-                setTimeout(resolve, TEST_TIMEOUT - 1000);
+                timer = setTimeout(resolve, TEST_TIMEOUT - 1000);
               }),
               fn!({path, run, source}),
             ]);
+            timer && clearTimeout(timer);
```

Comment:
```suggestion
            ]).finally(() => {
              if (timer) {
                clearTimeout(timer);
              }
            });
```
Otherwise it wont be cleared if the promise is rejected.

Reply:
Very thorough. Thx